### PR TITLE
PDA 197 - Create ProcessDefinition widget

### PR DESCRIPTION
**Pull request by gReis89**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **werbth**

If they are not already assigned, you can assign the PR to them by writing `/assign @werbth` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-process-driven-plugin).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-process-driven-plugin/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

File: widgets/src/components/ProcessDefinition/ProcessDefinitionConfig.js
Comment (rimselis) [inline widgets/src/components/ProcessDefinition/ProcessDefinitionConfig.js]: needed?
```suggestion
```

File: widgets/src/components/ProcessDefinition/ProcessDefinitionContainer.js
Comment (rimselis) [inline widgets/src/components/ProcessDefinition/ProcessDefinitionContainer.js]: `i18next`

File: widgets/src/components/ProcessDefinition/ProcessDefinitionContainer.js
Comment (rimselis) [inline widgets/src/components/ProcessDefinition/ProcessDefinitionContainer.js]: `i18next`

File: widgets/src/components/ProcessDefinition/ProcessDefinitionContainer.js
Comment (rimselis) [inline widgets/src/components/ProcessDefinition/ProcessDefinitionContainer.js]: `i18next`

File: widgets/src/components/ProcessDefinition/ProcessDefinitionContainer.js
Comment (rimselis) [inline widgets/src/components/ProcessDefinition/ProcessDefinitionContainer.js]: `i18next`

**Comment by entando-service** [comment]
All is well

**Review by rimselis — DISMISSED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-process-driven-plugin/pull/110#pullrequestreview-398265518" title="Approved">rimselis</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **werbth**

If they are not already assigned, you can assign the PR to them by writing `/assign @werbth` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-process-driven-plugin).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-process-driven-plugin/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **werbth**

If they are not already assigned, you can assign the PR to them by writing `/assign @werbth` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-process-driven-plugin).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-process-driven-plugin/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **werbth**

If they are not already assigned, you can assign the PR to them by writing `/assign @werbth` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-process-driven-plugin).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-process-driven-plugin/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by rimselis — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-process-driven-plugin/pull/110#pullrequestreview-398280869" title="Approved">rimselis</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **werbth**

If they are not already assigned, you can assign the PR to them by writing `/assign @werbth` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-process-driven-plugin).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-process-driven-plugin/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-service** [comment]
All is well

**Pull request merged**
